### PR TITLE
[IN-1550] fix install of cocoapods and weekly cache update

### PR DESCRIPTION
**Pull request by lmesz-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/ruby/tasks/ruby.yml [hunk 1]
```diff
@@ -29,6 +29,9 @@
 - name: CocoaPods specs install with pod
   shell: bash -l -c "pod repo add master https://github.com/CocoaPods/Specs.git"
   when: is_incremental_setup|default(false) == false and not lookup('env', 'IS_CI')
+  ignore_errors: true
```
Comment (bitrisekristof) [inline roles/ruby/tasks/ruby.yml:32]: Uhh somehow can we make sure it will be deleted after the fix?

Comment (lmesz-bitrise) [reply roles/ruby/tasks/ruby.yml:32]: You mean some kind of reminders or something like this?

Comment (bitrisekristof) [reply roles/ruby/tasks/ruby.yml:32]: YEs yes. Some reminder.

Comment (lmesz-bitrise) [reply roles/ruby/tasks/ruby.yml:32]: I set up a reminder in slack to notify the infra team about it on 10th of August

**Pull request merged**